src/Services/Import/CSV/CSVSettings.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Services\Import\CSV;
+
+class CSVSettings
+{
+    public function __construct(
+        private string $delimiter = ',',
+        private string $enclosure = ' ',
+        private string $escape = ' ',
+        private bool $haveHeader = false,
+    )
+    {
+    }
```

Comment:
because in the arguments of function I declare the fields of the class, and I initiate them immediately, and method haven't buisness logic. Or is it still better to create setters and initiate the object by them?